addOns/pscanrules/src/main/java/org/zaproxy/zap/extension/pscanrules/CsrfCountermeasuresScanRule.java

```diff
@@ -135,6 +136,7 @@ public void scanHttpResponseReceive(HttpMessage msg, int id, Source source) {
                                 + formElement.getParentElement()
                                 + "]");
                 StringBuilder sbForm = new StringBuilder();
+                List<String> formNames = new ArrayList<String>();
```

Comment:
Could use a sorted collection, it would be more clear the intent.

Reply:
Do you have a suggestion for the collection? A TreeSet works nicely but combines duplicate form field names into a single entry in the resulting output list, which I don't mind, seems better, but it is a change.

Reply:
Sounds good to me, I don't think that's a problem.

Reply:
Okay, I'll make the update, and add that to the CHANGELOG.